src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -764,5 +767,69 @@ internal IEnumerable<string> GetCodeSnippets()
 
             return (text.TrimEnd(endChars), restText.TrimEnd(endChars));
         }
+
+        internal void MessageInlineLink_Click(object sender, RoutedEventArgs e)
+        {
+            ThreadHelper.ThrowIfNotOnUIThread();
+            if (!(sender is XamlDoc.Hyperlink hyperLink))
+            {
+                return;
+            }
+
+            if (hyperLink.Tag is int id)
+            {
+                // The user clicked an in-line link with an integer target. Look for a Location object
+                // whose Id property matches that integer. The spec says that might be _any_ Location
+                // object under the current result. At present, we only support Location objects that
+                // occur in Result.Locations or Result.RelatedLocations. So, for example, we don't
+                // look in Result.CodeFlows or Result.Stacks.
+                LocationModel location =
+                    this.RelatedLocations.
+                    Concat(this.Locations).
+                    FirstOrDefault(l => l.Id == id);
```

Comment:
this wrap is strange. pls review it.
normally the `.` is in the start.

try to use the default wrapping from VS

Reply:
used VS wrapping format